python/libcudf/CMakeLists.txt

```diff
@@ -22,6 +22,8 @@ project(
   LANGUAGES CXX
 )
 
+option(USE_CUDA_NVCOMP_WHEEL "Use the CUDA nvcomp wheel instead of the system library" OFF)
```

Comment:
Here since it's documenting the behavior it's certainly helpful to make it explicit that the wheel is being used at runtime.